### PR TITLE
Adjust documentation after orchestration changes

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Adjusted documentation after orchestration changes

**Related issue(s)**

### CODE REVIEW

File: docs/kyma-environment-broker/03-10-orchestration.md [hunk 1]
```diff
@@ -3,16 +3,18 @@ title: Orchestration
 type: Details
 ---
 
-Orchestration is a mechanism that allows you to upgrade Kyma Runtimes. To create an orchestration, [send the upgrade request to the proper KEB handler](#tutorials-upgrade-kyma-runtime-using-keb). After sending the request, the orchestration is processed by `KymaUpgradeManager`. It lists Shoots (Kyma Runtimes) in the Gardener cluster and filters the Runtime IDs that you have specified in the request body. Then, `KymaUpgradeManager` performs the [upgrade steps](#details-runtime-operations) logic on the Runtimes that you have specified.
+Orchestration is a mechanism that allows you to upgrade Kyma Runtimes. To create an orchestration [fpllow this tutorial](#tutorials-upgrade-kyma-runtime-using-keb). After sending the request, the orchestration is processed by `KymaUpgradeManager`. It lists Shoots (Kyma Runtimes) in the Gardener cluster and filters the Runtime IDs that you have specified in the request body. Then, `KymaUpgradeManager` performs the [upgrade steps](#details-runtime-operations) logic on the Runtimes that you have specified.
```
Comment (crabtree) [inline docs/kyma-environment-broker/03-10-orchestration.md:6]:
```suggestion
Orchestration is a mechanism that allows you to upgrade Kyma Runtimes. To create an orchestration [follow this tutorial](#tutorials-upgrade-kyma-runtime-using-keb). After sending the request, the orchestration is processed by `KymaUpgradeManager`. It lists Shoots (Kyma Runtimes) in the Gardener cluster and narrows them to the IDs that you have specified in the request body. Then, `KymaUpgradeManager` performs the [upgrade steps](#details-runtime-operations) logic on the selected Runtimes.
```

Comment (crabtree) [inline docs/kyma-environment-broker/03-10-orchestration.md:6]: The links seem to be broken.

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 2]
```diff
@@ -13,21 +13,22 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
 
->**NOTE:** The **dry** parameter specified in the request body is set to `true`. As a result, the upgrade is executed in a fake mode. It means that all actions are not executed against selected Runtimes but the Orchestration status is still available.
+- target - can be used to select all runtimes by specifying it as `target: "all"`
+- globalAccount
+- subAccount
+- runtimeID
+- planName
+- region
+
+>**NOTE:** The **dryRun** parameter specified in the request body is set to `true`. As a result, the upgrade is executed in a fake mode. It means that all actions are executed against selected Runtimes but the upgrade process is not sending the upgrade request to Provisioner..
```
Comment (crabtree) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:31]: That is a bit confusing. I'd rephrase it
```suggestion
>**NOTE:** If the **dryRun** parameter specified in the request body is set to `true`, the upgrade is executed but the process is not sending the upgrade request to the Provisioner.
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 3]
```diff
@@ -36,19 +37,54 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
    --data-raw "{
```
Comment (crabtree) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:37]: I think this should look like this:
```bash
curl --request POST "https://$BROKER_URL/upgrade/kyma" \
--header "$AUTHORIZATION_HEADER" \
--header 'Content-Type: application/json' \
--data-raw "{\
    \"targets\": {\
        \"include\": {\
            \"runtimeID\": \"uuid-sdasd-sda23t-efs\",\
            \"globalAccount\": \"uuid-sdasd-sda23t-efs\",\
            \"subAccount\": \"uuid-sdasd-sda23t-efs\",\
            \"planName\": \"azure\",\
            \"region\": \"europewest\",\
         },\
    },\
    \"dryRun\": false\
}"
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 4]
```diff
@@ -36,19 +37,54 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
-4. [Check the operation status](#tutorials-check-orchestration-status).
+4. [Check the orchestration status](#tutorials-check-orchestration-status).
+
+>**NOTE:** Only one orchestration can be processed at the same time. If the KEB is already processing some orchestration the newly created will wait for processing with the **pending** state.
+
+### Strategies
+
+To change the behaviour of the orchestration you can specify a strategy in the request body.
+
+For now we support only the **Parallel** type of strategy with two types of schedule:
```
Comment (crabtree) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:69]:
```suggestion
For now we support only the **Parallel** strategy with two types of schedule:
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 5]
```diff
@@ -36,19 +37,54 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+
+- Immediate - schedules the upgrade operations instantly
+- MaintenanceWindow - schedules the upgrade operations in the time windows specified for each runtime
```
Comment (crabtree) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:72]:
```suggestion
- MaintenanceWindow - schedules the upgrade operations respecting the maintenance time windows specified for given runtime
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 6]
```diff
@@ -36,19 +37,54 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+
+You can also configure how many upgrade operations can be executed in parallel to accelerate the process by specifying the **parallel** object in the request body with field **workers** set to number of the desired concurrent executions of the upgrade operations.
```
Comment (crabtree) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:74]:
```suggestion
You can also configure how many upgrade operations can be executed in parallel to accelerate the process. Specify the **parallel** object in the request body with **workers** field set to the number of concurrent executions for the upgrade operations.
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 7]
```diff
@@ -36,19 +37,54 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+    "type": "parallel",
+    "schedule": "maintenanceWindow",
+    "parallel": {
+      "workers": 5
+    }
+  }
+}
+```
+
+>**NOTE:** Without specifying the **strategy** field, the orchestration will be executed with Parallel strategy, using the immediate type of schedule with only one worker.
```
Comment (crabtree) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:90]:
```suggestion
>**NOTE:** By default the orchestration will be executed with Parallel strategy, using the immediate type of schedule with only one worker.
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 8]
```diff
@@ -36,19 +37,54 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+               \"planName\": "azure",
+               \"region\": "europewest",
             },
        },
-        \"dry\": false,
+       \"dryRun\": false
    }"
    ```
 
+>**NOTE:** Without specifying the **targets** field, the orchestration will select all the runtimes by default using the **all** target.
```
Comment (crabtree) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:51]:
```suggestion
>**NOTE:** By default, the orchestration will select all the runtimes using the **all** target.
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 9]
```diff
@@ -72,38 +63,120 @@ Make a call to the Kyma Environment Broker with a proper **Authorization** [requ
...
+## Fetch the detailed single operation
+
+1. Export the following values as the environment variables:
+
+   ```bash
+   export ORCHESTRATION_ID={OBTAINED_ORCHESTRATION_ID}
+   export OPERATION_ID={SCHEDULED_OPERATION_ID}
+   ```
+
+2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-authorization) to verify that the orchestration succeeded.
```
Comment (crabtree) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:154]:
```suggestion
2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-authorization) to fetch the details for given operation.
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 10]
```diff
@@ -72,38 +63,120 @@ Make a call to the Kyma Environment Broker with a proper **Authorization** [requ
...
+
+## List the upgrade operations
+
+1. Export the orchestration ID that you obtained during the upgrade call as an environment variable:
+
+   ```bash
+   export ORCHESTRATION_ID={OBTAINED_ORCHESTRATION_ID}
+   ```
+
+2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-authorization) to verify that the orchestration succeeded.
```
Comment (crabtree) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:102]:
```suggestion
2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-authorization) to fetch the list of the upgrade operations for a given orchestration.
```

**Review by crabtree — CHANGES_REQUESTED** [review]
Please see my comments.

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md (hunk 8 above)
Comment (polskikiel) [reply docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md]: Will remove that sentence because implementation has changed after a review

File: docs/kyma-environment-broker/03-10-orchestration.md (hunk 1 above)
Comment (polskikiel) [reply docs/kyma-environment-broker/03-10-orchestration.md]: it will work on our website.

**Review by crabtree — APPROVED** [review]

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 11]
```diff
@@ -13,42 +13,76 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+    "type": "parallel",
+    "schedule": "maintenanceWindow",
+    "parallel": {
+      "workers": 5
+    }
+  }
+}
+```
+
+>**NOTE:** By default the orchestration will be executed with Parallel strategy, using the immediate type of schedule with only one worker.
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:88]:
```suggestion
>**NOTE:** By default, the orchestration is executed with the parallel strategy, using the immediate type of schedule with only one worker.
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 12]
```diff
@@ -13,42 +13,76 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+       "orchestrationID":"8a7bfd9b-f2f5-43d1-bb67-177d2434053c"
    }
    ```
 
-4. [Check the operation status](#tutorials-check-orchestration-status).
+4. [Check the orchestration status](#tutorials-check-orchestration-status).
+
+>**NOTE:** Only one orchestration can be processed at the same time. If the KEB is already processing some orchestration the newly created will wait for processing with the **pending** state.
+
+### Strategies
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:63]: Shouldn't this section be a part of the general document about orchestration? This is a tutorial and it's not a part of the steps to perform. Let's discuss.

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 13]
```diff
@@ -13,42 +13,76 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+
+To change the behaviour of the orchestration you can specify a strategy in the request body.
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:65]:
```suggestion
To change the behavior of the orchestration, you can specify a strategy in the request body.
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 14]
```diff
@@ -13,42 +13,76 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:66]:
```suggestion
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 15]
```diff
@@ -13,42 +13,76 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+For now we support only the **Parallel** strategy with two types of schedule:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:67]:
```suggestion
For now, we support only the **parallel** strategy with two types of schedule:
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 16]
```diff
@@ -13,42 +13,76 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+
+- Immediate - schedules the upgrade operations instantly
+- MaintenanceWindow - schedules the upgrade operations respecting the maintenance time windows specified for given runtime
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:70]:
```suggestion
- MaintenanceWindow - schedules the upgrade operations with the maintenance time windows specified for a given Runtime
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 17]
```diff
@@ -13,42 +13,76 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+
+You can also configure how many upgrade operations can be executed in parallel to accelerate the process. Specify the **parallel** object in the request body with **workers** field set to the number of concurrent executions for the upgrade operations.
+
+The example strategy objects looks as follows:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:74]:
```suggestion
The example strategy object looks as follows:
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 18]
```diff
@@ -3,10 +3,11 @@ title: Check orchestration status
...
 
-This tutorial shows how to check the orchestration status for the cluster and Kyma orchestrations. You can either:
-- Fetch a single orchestration.
-- Fetch all orchestrations.
-
+This tutorial shows how to check the orchestration status for the cluster and Kyma orchestrations. Using this API you can fetch data about:
+- A single orchestration
+- All orchestrations
+- Upgrade operations scheduled by a given orchestration
+- A single operation with details like parameters send to the Provisioner
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:10]:
```suggestion
- A single operation with details, such as parameters sent to Runtime Provisioner
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 19]
```diff
@@ -3,10 +3,11 @@ title: Check orchestration status
...
 
 ## Fetch a single orchestration
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:12]:
```suggestion
## Fetch a single orchestration status
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 20]
```diff
@@ -3,10 +3,11 @@ title: Check orchestration status
 type: Tutorials
 ---
 
-This tutorial shows how to check the orchestration status for the cluster and Kyma orchestrations. You can either:
-- Fetch a single orchestration.
-- Fetch all orchestrations.
-
+This tutorial shows how to check the orchestration status for the cluster and Kyma orchestrations. Using this API you can fetch data about:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:6]:
```suggestion
This tutorial shows how to check the orchestration status. Using this API you can fetch data about:
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 21]
```diff
@@ -19,46 +20,36 @@ This tutorial shows how to check the orchestration status for the cluster and Ky
...
+            "parallel": {
+                "workers": 1
+            }
+        },
+        "dryRun": true
     }
+}
    ```
 
 ## Fetch all orchestrations
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:55]:
```suggestion
## Fetch all orchestrations status
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 22]
```diff
@@ -72,38 +63,120 @@ Make a call to the Kyma Environment Broker with a proper **Authorization** [requ
...
-        [...]
-      ]
-    }]
+    ],
+    "count": 2,
+    "totalCount": 2
+}
+   ```
+
+## Fetch the detailed single operation
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:145]:
```suggestion
## Fetch the detailed operation status
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 23]
```diff
@@ -72,38 +63,120 @@ Make a call to the Kyma Environment Broker with a proper **Authorization** [requ
...
+              }
+          },
+          "dryRun": true
       }
-      "runtimeOperations": [
+  }
+]
+   ```
+
+## List the upgrade operations
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:94]:
```suggestion
## List upgrade operations
```

Comment (klaudiagrz) [reply docs/kyma-environment-broker/08-06-orchestration-status.md:94]: Shouldn't this heading match the bullet point from the overview section?

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 24]
```diff
@@ -72,38 +63,120 @@ Make a call to the Kyma Environment Broker with a proper **Authorization** [requ
 A successful call returns a list of all orchestrations:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:63]:
```suggestion
A successful call returns the list of all orchestrations:
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 25]
```diff
@@ -72,38 +63,120 @@ Make a call to the Kyma Environment Broker with a proper **Authorization** [requ
...
+   export ORCHESTRATION_ID={OBTAINED_ORCHESTRATION_ID}
+   ```
+
+2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-authorization) to fetch the list of the upgrade operations for a given orchestration.
+
+   ```bash
+   curl --request GET "https://$BROKER_URL/orchestrations/$ORCHESTRATION_ID/operations --header "$AUTHORIZATION_HEADER""
+   ```
+
+A successful call returns a list of the upgrade operations:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:108]:
```suggestion
A successful call returns the list of upgrade operations:
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 26]
```diff
@@ -72,38 +63,120 @@ Make a call to the Kyma Environment Broker with a proper **Authorization** [requ
...
+   export OPERATION_ID={SCHEDULED_OPERATION_ID}
+   ```
+
+2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-authorization) to fetch the details for given operation.
+
+   ```bash
+   curl --request GET "https://$BROKER_URL/orchestrations/$ORCHESTRATION_ID/operations/$OPERATION_ID --header "$AUTHORIZATION_HEADER""
+   ```
+
+A successful call returns the upgrade operation object with `kymaConfig` and `clusterConfig` fields:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:160]:
```suggestion
A successful call returns the upgrade operation object with the **kymaConfig** and **clusterConfig** fields:
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 27]
```diff
@@ -13,42 +13,76 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
 
    ```bash
    export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
    ```
 
-3. Make a call to the Kyma Environment Broker to orchestrate the upgrade.
+2. Make a call to the Kyma Environment Broker to orchestrate the upgrade. You can select runtimes to upgrade using a following selectors:
 
->**NOTE:** The **dry** parameter specified in the request body is set to `true`. As a result, the upgrade is executed in a fake mode. It means that all actions are not executed against selected Runtimes but the Orchestration status is still available.
+- target - can be used to select all runtimes by specifying it as `target: "all"`
```
[the remark this answers is not in the document]
Comment (polskikiel) [reply docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:24]: I will code everything with proper descriptions

File: docs/kyma-environment-broker/08-06-orchestration-status.md (hunk 23 above)
Comment (polskikiel) [reply docs/kyma-environment-broker/08-06-orchestration-status.md]: we could go with `List upgrade operations scheduled by orchestration` but imo it's a bit too long like for a heading

File: docs/kyma-environment-broker/03-10-orchestration.md [hunk 28]
```diff
@@ -3,16 +3,42 @@ title: Orchestration
...
+- `GET /orchestrations` - exposes data about all orchestrations.
+- `GET /orchestrations/{orchestration_id}` - exposes data about a single orchestration status.
+- `GET /orchestrations/{orchestration_id}/operations` - exposes data about operations scheduled by the orchestration with a given ID.
+- `GET /orchestrations/{orchestration_id}/operations/{operation_id}` - exposes the detailed data about a single operation with a given ID.
 - `POST /upgrade/kyma` - schedules the orchestration. It requires specifying a request body.
 
-For more details about the API, check the [Swagger schema](https://app.swaggerhub.com/apis/kempski/kyma-orchestration_api/0.1).
+For more details about the API, check the [Swagger schema](https://app.swaggerhub.com/apis/kempski/kyma-orchestration_api/0.4).
+
+### Strategies
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-10-orchestration.md:22]:
```suggestion
## Strategies
```

File: docs/kyma-environment-broker/03-10-orchestration.md [hunk 29]
```diff
@@ -3,16 +3,42 @@ title: Orchestration
...
+
+To change the behavior of the orchestration, you can specify a strategy in the request body.
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-10-orchestration.md:24]:
```suggestion
To change the behavior of the orchestration, you can specify a **strategy** in the request body.
```

File: docs/kyma-environment-broker/03-10-orchestration.md [hunk 30]
```diff
@@ -3,16 +3,42 @@ title: Orchestration
...
+For now, we support only the **parallel** strategy with two types of schedule:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-10-orchestration.md:25]:
```suggestion
For now, there is only one **parallel** strategy with two types of schedule:
```

File: docs/kyma-environment-broker/03-10-orchestration.md [hunk 31]
```diff
@@ -3,16 +3,42 @@ title: Orchestration
...
+
+- Immediate - schedules the upgrade operations instantly
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-10-orchestration.md:27]:
```suggestion
- Immediate - schedules the upgrade operations instantly.
```

File: docs/kyma-environment-broker/03-10-orchestration.md [hunk 32]
```diff
@@ -3,16 +3,42 @@ title: Orchestration
...
+- MaintenanceWindow - schedules the upgrade operations with the maintenance time windows specified for a given Runtime
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-10-orchestration.md:28]:
```suggestion
- MaintenanceWindow - schedules the upgrade operations with the maintenance time windows specified for a given Runtime.
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 33]
```diff
@@ -13,42 +13,80 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
-1. Export these values as environment variables:
+1. [Get the access token](#details-authorization). Export this variable based on the token you got from the OAuth client:
 
    ```bash
-   export RUNTIME_ID={RUNTIME_ID}
+   export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
    ```
 
-2. [Get the access token](#details-authorization). Export this variable based on the token you got from the OAuth client:
+2. Make a call to the Kyma Environment Broker to orchestrate the upgrade. You can select Runtimes to upgrade using the following selectors:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:22]:
```suggestion
2. Make a call to the Kyma Environment Broker to orchestrate the upgrade. You can select specific Runtimes to upgrade using the following selectors:
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 34]
```diff
@@ -13,42 +13,80 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+
+```
+- target - can be used to select all runtimes by specifying it as `target: "all"`
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:25]:
```suggestion
- `target` - use the `target: "all"` selector to selects all Runtimes
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 35]
```diff
@@ -13,42 +13,80 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+- globalAccount - selects runtimes with specified globalAccount ID
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:26]:
```suggestion
- `globalAccount` - use it to select Runtimes with the specified global account ID
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 36]
```diff
@@ -13,42 +13,80 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+- subAccount - selects runtimes with specified subAccount ID
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:27]:
```suggestion
- `subAccount` - use it to select Runtimes with the specified subaccount ID
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 37]
```diff
@@ -13,42 +13,80 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+- runtimeID - selects runtimes with specified runtime ID
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:28]:
```suggestion
- `runtimeID` - use it to select Runtimes with the specified Runtime ID
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 38]
```diff
@@ -13,42 +13,80 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+- planName - selects runtimes with specified plan name
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:29]:
```suggestion
- `planName` - use it to select Runtimes with the specified plan name
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 39]
```diff
@@ -13,42 +13,80 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+- region - selects runtimes located in specified region
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:30]:
```suggestion
- `region` - use it to select Runtimes located in the specified region
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 40]
```diff
@@ -13,42 +13,80 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
 
    ```bash
-   export RUNTIME_ID={RUNTIME_ID}
+   export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
    ```
 
-2. [Get the access token](#details-authorization). Export this variable based on the token you got from the OAuth client:
+2. Make a call to the Kyma Environment Broker to orchestrate the upgrade. You can select Runtimes to upgrade using the following selectors:
+
+```
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:24]:
```suggestion
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 41]
```diff
@@ -13,42 +13,80 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+- target - can be used to select all runtimes by specifying it as `target: "all"`
+- globalAccount - selects runtimes with specified globalAccount ID
+- subAccount - selects runtimes with specified subAccount ID
+- runtimeID - selects runtimes with specified runtime ID
+- planName - selects runtimes with specified plan name
+- region - selects runtimes located in specified region
+```
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:31]:
```suggestion
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 42]
```diff
@@ -13,42 +13,80 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
+            \"planName\": \"azure\",\
+            \"region\": \"europewest\",\
+         },\
+    },\
+    \"dryRun\": false\
+}"
    ```
 
-3. Make a call to the Kyma Environment Broker to orchestrate the upgrade.
+>**NOTE:** If the **dryRun** parameter specified in the request body is set to `true`, the upgrade is executed but the upgrade request is not sent to the Runtime Provisioner.
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:51]:
```suggestion
>**NOTE:** If the **dryRun** parameter specified in the request body is set to `true`, the upgrade is executed but the upgrade request is not sent to Runtime Provisioner.
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 43]
```diff
@@ -13,42 +13,80 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
 
->**NOTE:** The **dry** parameter specified in the request body is set to `true`. As a result, the upgrade is executed in a fake mode. It means that all actions are not executed against selected Runtimes but the Orchestration status is still available.
+3. To configure strategy of your orchestration, use the following request example:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:53]:
```suggestion
3. If you want to configure [the strategy of your orchestration](#details-orchestration-strategies), use the following request example:
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 44]
```diff
@@ -3,12 +3,13 @@ title: Check orchestration status
 type: Tutorials
 ---
 
-This tutorial shows how to check the orchestration status for the cluster and Kyma orchestrations. You can either:
-- Fetch a single orchestration.
-- Fetch all orchestrations.
+This tutorial shows how to check the orchestration status. Using this API you can fetch data about:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:6]:
```suggestion
This tutorial shows how to check the orchestration status. Using this API, you can fetch data about:
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 45]
```diff
@@ -19,91 +20,163 @@ This tutorial shows how to check the orchestration status for the cluster and Ky
...
+              }
+          },
+          "dryRun": true
       }
-      "runtimeOperations": [
+  }
+]
+   ```
+
+## List upgrade operations scheduled by orchestration
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:94]:
```suggestion
## List upgrade operations scheduled by an orchestration
```

File: docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md [hunk 46]
```diff
@@ -13,42 +13,78 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
...
 
    ```bash
    export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
    ```
 
-3. Make a call to the Kyma Environment Broker to orchestrate the upgrade.
+2. Make a call to the Kyma Environment Broker to orchestrate the upgrade. You can select specific Runtimes to upgrade using the following selectors:
 
->**NOTE:** The **dry** parameter specified in the request body is set to `true`. As a result, the upgrade is executed in a fake mode. It means that all actions are not executed against selected Runtimes but the Orchestration status is still available.
+- `target` - use the `target: "all"` selector to selects all Runtimes
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md:24]:
```suggestion
- `target` - use the `target: "all"` selector to select all Runtimes
```

**Review by klaudiagrz — APPROVED** [review]
LGTM 🦊

**Review by crabtree — APPROVED** [review]
